03/app.test.js

```diff
@@ -1 +1,29 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('throw error if "min" is not a number', () => {
+    expect( () => randomNumber('1', 2) ).toThrow();
```

Comment:
Jasne, ale trzeba napisać jakiego błędu się spodziewasz - inaczej nie wiesz czy faktycznie to działa jak trzeba ;)